### PR TITLE
add tip to store all config in VCS before pushing to cluster

### PR DESCRIPTION
**Pull request by mikedanese**
@lavalamp @davidopp

### CODE REVIEW

File: docs/user-guide/config-best-practices.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Documentation for other releases can be found at
 This document is meant to highlight and consolidate in one place configuration best practices that are introduced throughout the user-guide and getting-started documentation and examples. This is a living document so if you think of something that is not on this list but might be useful to others, please don't hesitate to file an issue or submit a PR.
 
 1. When writing configuration, use the latest stable API version (currently v1). 
+1. Configuration should be stored in version control before getting pushed to the cluster. This allows configuration to be quickly rolled back if needed and will aid with cluster recreation and restoration if the worst were to happen.
```
Comment (lavalamp) [inline docs/user-guide/config-best-practices.md]: s/getting/being/

**Comment by lavalamp** [comment]
LGTM

File: docs/user-guide/config-best-practices.md (hunk 1 above)
Comment (davidopp) [inline docs/user-guide/config-best-practices.md]: recreation -> re-creation

Comment (mikedanese) [inline docs/user-guide/config-best-practices.md]: fixed both

**Comment by davidopp** [comment]
LGTM

We've been merging a bunch of stuff without letting shippable run but I'd really like to let it run on this PR because @timstclair says there are still problems

**Comment by mikedanese** [comment]
@davidopp flaky e2e test but shippable an travis passed

**Comment by bgrant0607** [comment]
It looks to me like the breakage wasn't due to not running shippable, but due to not re-running checks after each merge.

**Pull request merged**